guides/training_with_built_in_methods.py

```diff
@@ -43,7 +43,7 @@
 ## API overview: a first end-to-end example
 
 When passing data to the built-in training loops of a model, you should either use
-**NumPy arrays** (if your data is small and fits in memory) or **`tf.data Dataset`
+**NumPy arrays** (if your data is small and fits in memory) or **[tf.data.Dataset](https://www.tensorflow.org/api_docs/python/tf/data/Dataset?version=nightly)
```

Comment:
You can remove the link. It will be autogenerated.

Reply:
Hi @fchollet, Updated!